model/src/main/java/com/javahelp/model/user/User.java

```diff
@@ -0,0 +1,86 @@
+package com.javahelp.model.user;
+
+/**
+ * A class for a user entity
+ */
+public class User {
+
+    /**
+     * The ID of the user account (randomly generated and immutable)
+     */
+    private String stringID;
+
+    /**
+     * The information of the current user
+     */
+    private UserInfo userInfo;
+
+    /**
+     * The username of the current user
+     */
+    private String username;
+
+    /**
+     * A constructor for the current User Entity
+     *
+     * @param stringID the ID of the account
+     * @param userInfo the account information
+     * @param username the username of the account (randomly generated)
+     */
+    public User(String stringID, UserInfo userInfo, String username) {
+        this.stringID = stringID;
+        this.userInfo = userInfo;
+        this.username = username;
+    }
+
+    /**
+     * Set the StringID for the user
+     *
+     * @param stringID an updated ID of the account
+     */
+    public void setStringID(String stringID) {
```

Comment:
If we cannot change the stringID, we probably don't need a setter for this attribute.